install/scripts/post-install.sh

```diff
@@ -21,12 +21,3 @@ chown -R vagrant:vagrant "$HOME_DIR"
 # Cycle tomcat
 cd /var/lib/tomcat7
 service tomcat7 restart
-
-# Cycle karaf and watch the maven bundles
-service karaf-service restart
```

Comment:
Along with keeping the `fcrepo-indexing-triplestore` service above, we might want to re-add this restart.